release.toml

```diff
@@ -10,9 +11,13 @@ pre-release-replacements = [
   {file="../CHANGELOG.adoc", search="// cargo-release: next header goes here \\(do not change this line\\)", replace="// cargo-release: next header goes here (do not change this line)\n\n== Unreleased changes (release date TBD)\n\nhttps://github.com/oxidecomputer/dropshot/compare/{{tag_name}}\\...HEAD[Full list of commits]", exactly=1},
 ]
 
-disable-push = true
-pre-release-commit-message = "release {{crate_name}} {{version}}"
-post-release-commit-message = "starting {{crate_name}} {{next_version}} after releasing {{version}}"
-tag-message = "release {{crate_name}} {{version}}"
+push = false
+pre-release-commit-message = "release {{version}}"
+post-release-commit-message = "starting {{next_version}} after releasing {{version}}"
+tag-message = "release {{version}}"
 tag-prefix=""
+dev-version = true
 dev-version-ext = "dev"
+allow-branch = [ "main" ]
+shared-version = true
```

Comment:
this was the item that I alluded to earlier; obviously you already figured it out!